config/locales/en.yml

```diff
@@ -38,3 +7,181 @@ en:
           attributes:
             date_of_birth:
               before: "must be before 18 years ago"
+            last_name:
```

Comment:
Not necessary to duplicate these localizations. You can find it in ActiveRecord gem.
```ruby
 last_name:
              blank: "can't be blank"
              too_short: "is too short (minimum is %{count} characters)"
              too_long: "is too long (maximum is %{count} characters)"
            first_name:
              blank: "can't be blank"
              too_short: "is too short (minimum is %{count} characters)"
              too_long: "is too long (maximum is %{count} characters)"
            middle_name:
              blank: "can't be blank"
              too_short: "is too short (minimum is %{count} characters)"
              too_long: "is too long (maximum is %{count} characters)"
            passport_data:
              blank: "can't be blank"
              is: "is the wrong length (should be %{count} characters)"
            date_of_birth:
              blank: "can't be blank"
              before: "must be before %{restriction}"
            place_of_birth:
              blank: "can't be blank"
              invalid: "is invalid"
            home_address:
              blank: "can't be blank"
              invalid: "is invalid
```